packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -226,17 +228,27 @@ class DaemonDomain extends Domain {
     registerHandler('shutdown', shutdown);
 
     _subscription = daemon.notifyingLogger.onMessage.listen((LogMessage message) {
-      if (message.stackTrace != null) {
-        sendEvent('daemon.logMessage', <String, dynamic>{
-          'level': message.level,
-          'message': message.message,
-          'stackTrace': message.stackTrace.toString()
-        });
+      if (daemon.logToStdout) {
+        if (message.level == 'status') {
+          print(message.message);
+        } else if (message.level == 'error') {
+          stderr.writeln(message.message);
```

Comment:
It's asymmetric. I would like the first statement to read `stdout.writeln` - for symmetry with the next one - but I need it to be a print(), so we can capture print output in a zone when running tests, in order to test this functionality.